DOCSP-28958 Clarify Typescript Find and the _id section (#644)
mongodb/docs-node: source/fundamentals/typescript.txt

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The ``find`` and ``findOne`` methods of the ``Collection`` class include
the ``_id`` field in their return type. The driver infers the type of the
returned ``_id`` field based on the type parameter you passed to your
``Collection`` instance.

If the type parameter you passed to your ``Collection`` instance includes the
``_id`` field in its schema, the driver infers that the ``_id`` field returned
from the method is of the type specified in the schema.

However, if the type parameter you passed to your ``Collection`` instance does not
include the ``_id`` field in its schema, the driver infers that the type of the
``_id`` field returned from the method is ``ObjectId``. 

.. tip::

   The type parameter passed to your ``Collection`` influences only the type
   inference of the fields returned from the method. The driver does not convert

>    the field to the specified type. The type of the fields in your type
>    parameter's schema should match the type of their corresponding fields in the
>    collection.

Reviewer: S: Slight ambiguity

```suggestion
   the field to the specified type. The type of each field in your type
   parameter's schema should match the type of the corresponding field in the
   collection.
```
